packages/documentation/src/pages/forms/form-tutorial-basic.mdx

```diff
@@ -1,25 +1,26 @@
 ---
-title: Vets Website Form Tutorial - Basic
+title: VAFS tutorial - basic
 ---
 
-# Form Tutorial - Basic
+# Basic tutorial
 
-In this doc, we'll walk through creating a new form and making some simple changes.
+In this doc, we'll walk through creating a new form and making some simple changes. 
 
-## Setting up
+Prerequisites: 
+- VA.gov [running locally](/getting-started)
+- [VA.gov Yeoman generator](/platform/generator) installed locally (first step in linked page)
+
+If you both of those steps completed, you're ready to get started.
 
-If you haven't already, get [vets-website](https://github.com/department-of-veterans-affairs/vets-website) set up and running on your machine.
+## Setting up
 
-The first step after that is to get the basic scaffolding set up for a new application. We have a [generator tool](https://github.com/department-of-veterans-affairs/vets-website/blob/master/docs/GeneratorOptions.md) to help you set up the files needed for the new React app and form. In the root of `vets-website`, you should install and run Yeoman and the generator:
+To begin, run the generator from the root of your local vets-website repository:
 
 ```bash
-npm install -g yo
-npm install -g @department-of-veterans-affairs/generator-vets-website
-
 yo @department-of-veterans-affairs/vets-website
 ```
 
-You can answer the questions with the [documentation](https://github.com/department-of-veterans-affairs/vets-website/blob/master/docs/GeneratorOptions.md) as a guide, or if you're just looking to try things out, here are the answers I'm using for this tutorial:
+You can answer the questions with the [usage documentation](/platform/generator) as a guide. This tutorial uses the following answers to the questions asked by the generator:
```

Comment:
Since this is a tutorial I think we should remove the options and tell people specifically what to do. 
- set up the yeoman generator following the guide 
- use these answers:

maybe put the question/ answers into a table